specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscCompilationJob.json

```diff
@@ -0,0 +1,608 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2018-01-15",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "syncMethods": "None"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/compilationjobs/{compilationJobName}": {
+      "put": {
+        "tags": [
+          "DscCompilationJob"
+        ],
+        "operationId": "DscCompilationJob_Create",
```

Comment:
>DscCompilationJob_Create [](start = 24, length = 24)

Is this a long running operation?

Reply:
No this is not. It just submits a job and returns the JobId, which is then used to track the progress of the Job.

Reply:
Should clients then poll for operation completion?

Reply:
Yes.

Reply:
Looks like a long running operation to me

Reply:
After [reading this](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#enable-asynchronous-operations-with-x-ms-long-running-operation) up, it makes sense to mark this as ` x-ms-long-running-operation`. I will do that.

Reply:
Done.

Reply:
Fixed